stable/xray/values.yaml

```diff
@@ -29,6 +29,8 @@ replicaCount: 1
 
 xray:
   name: xray
+  labels: {}
+  extraLabels: {}
```

Comment:
What is the different between `labels` and `extraLabels`? I don't understand why we need both